runtime/src/chpl-tasks.c

```diff
@@ -207,3 +208,11 @@ size_t chpl_task_getDefaultCallStackSize(void)
 
   return deflt;
 }
+
+void chpl_task_threadOnPark(void) {
+  chpl_qsbr_blocked();
+}
+
+void chpl_task_threadOnUnpark(void) {
+  chpl_qsbr_unblocked();
+}
```

Comment:
Let's add the newline at the end of the file

Reply:
Done.